grammars/javascript.cson

```diff
@@ -724,6 +724,41 @@
     ]
   }
   {
+    # Math
+    'begin': '\\bMath\\b'
```

Comment:
Sorry, what I meant was that it will successfully match `Math` as `support.class.js` and attempt to begin matching the subgroups.